.github/workflows/ci-windows.yml

```diff
@@ -75,7 +76,15 @@ jobs:
         run: cmake --build build --config Release --parallel 4
       - name: Run AVIF Tests
         working-directory: ./build
-        run: ctest -j $Env:NUMBER_OF_PROCESSORS --output-on-failure
+        run: ctest -C Release -j $Env:NUMBER_OF_PROCESSORS --output-on-failure
+      - name: Set the config folder for Ninja
+        if: ${{ matrix.generator == "Ninja" }}
+        run:
+          echo "CONFIG_FOLDER=." >> $env:GITHUB_ENV
```

Comment:
Nit: Use `_DIR` instead of `_FOLDER` in the variable name `CONFIG_FOLDER`. `_DIR` is more common in our code.

Reply:
Done